src/main.cpp

```diff
@@ -196,6 +196,8 @@ MainWindow *QTerminalApp::newWindow(bool dropMode, TerminalConfig &cfg)
     else
     {
         window = new MainWindow(cfg, dropMode);
+        if (Properties::Instance()->windowMaximized)
+            window->setWindowState(window->windowState() | Qt::WindowMaximized);
         window->show();
```

Comment:
`setWindowState` takes `Qt::WindowState` as its argument while you've given it `Qt::WindowStates`. Ref. http://doc.qt.io/qt-5/qwindow.html

Reply:
So, to make sure I get this right, the proper code looks like this?
```
window->setWindowStates(window->windowState() | Qt::WindowMaximized);
```

Reply:
To answer my own question, that would make sense, so I'll change it.

If I'm understanding this right, `Qt::WindowStates` can still take only one argument, so it wouldn't hurt either way to be safe here,

Reply:
My above comment may sound pedantic but future versions of Qt might not accept it. Please test `setWindowState(Qt::WindowMaximized);` it should be enough.

Reply:
Except, that's FTBFS...
```
src/main.cpp:200:21: error: ‘class MainWindow’ has no member named ‘setWindowStates’; did you mean ‘setWindowState’?
             window->setWindowStates(window->windowState() | Qt::WindowMaximized);
                     ^~~~~~~~~~~~~~~
                     setWindowState
```

Reply:
ACK

Reply:
My suggestion was `setWindowState(Qt::WindowMaximized)`.